### ISSUE TITLE
Filter issues by "subscribed by me".

### ISSUE THREAD

**Issue by buhtz** [dear-github/dear-github#247]
I can show issues reported by me. But I can not see a list of the issues I am subscribed to.

**Comment by tbergeron** [comment]
Not much thumbs up?! Really? This is definitely worth adding.

**Comment by samal84** [comment]
Also mentioned here: https://github.com/isaacs/github/issues/283

**Comment by IvanMalison** [comment]
Mentioning myself so I can find this later @IvanMalison

**Comment by gepcel** [comment]
+1 on this. And make a comment here so I can find it later. Otherwise I won't be able to find this by only `subscribe` this issue.

**Comment by lionlai1989** [comment]
But it wouldn't work if the issue has been cloesd.

**Comment by chuihinwai** [comment]
Please.

**Comment by DaveQB** [comment]
I always find it very hard to find Issues on github I have commented on and want to go back to them to update them. This gets my vote.
The mentioning myself work around i a good idea. @DaveQB

**Comment by rivy** [comment]
cc: @rivy 
Using "involves" can be a workaround, eg "involves:rivy".
Also, see a longer similar discussion at https://github.com/isaacs/github/issues/283 .

**Comment by RalfJung** [comment]
Also with "involves", if you commented but then unsubscribed, the issue is still listed. I would like a list I can actually curate, by unsubscribing from what is no longer relevant.

Maybe Github will have feature parity with Bugzilla -- one day. ;)

**Comment by frou** [comment]
Guess what... It's finally here! https://github.com/notifications/subscriptions

**Comment by xianshenglu** [comment]
Awesome!

**Comment by RalfJung** [comment]
> It's finally here!

Well, not really. There's a list (with 1000+ entries in my case, which is not very useful), but there does not seem to be a way to combine this with other search criteria -- like "only open issues/PRs I subscribed to", or adding free-text search.

**Comment by pavel-artanov** [comment]
Guys, really, many thanks! It's one of the awesome feature! I can't describe my feelings :-)

**Comment by clarkbw** [comment]
> "only open issues/PRs I subscribed to", or adding free-text search.

Can you add that as a new issue(s)?  I think "Subscribed by me", this issue, has been fixed.

**Comment by PeterDeathBredon** [comment]
It would be so helpful to have a filter like subscribed:@me in project boards and the issues list.

**Comment by PeterDeathBredon** [comment]
> It would be so helpful to have a filter like subscribed:@me in project boards and the issues list.

oh, I overlooked it in the documentation: involves:USERNAME does exactly what we need.

**Comment by aisbergde** [comment]
on Android I am using the github App. It is mostly required for authentication. But if the app is installed, it is hard to open github in web browser. As workarounds I use a second browser. But it is complicated. And the app is limited in many features.

When I am exploring the issues for a project, it would just be intuitive to be able use my subscription as an additional filter. Without the need to remember any syntax but just by using an intuitive option.

The list of notifications is fine, but it is a mix of all issues I subscribed. Hard to filter.

**Comment by fireattack** [comment]
> > It would be so helpful to have a filter like subscribed:@me in project boards and the issues list.
> 
> oh, I overlooked it in the documentation: involves:USERNAME does exactly what we need.

No, `involves:@me` filter does not include issues you merely subscribe to. 

~~But it looks that `subscribed:@me` does work at the moment (probably not back then).~~

~~Also note unlike `author:`, `involves:` etc., since this is "private" you have to use `@me`, not your actual username.~~

Edit: I was wrong, searching `subscribed:@me` will just return some random issues.. not the ones I actually subscribed. A bummer. So the issue remains unsolved :(

**Comment by mfn** [comment]
Small comfort: in case you commented on the issue, you can use the filter `commenter:@me`. Better then nothing…

**Comment by nair-sumesh** [comment]
> Small comfort: in case you commented on the issue, you can use the filter `commenter:@me`. Better then nothing…

Did not work for me

**Comment by colonelpanic8** [comment]
sooooo crazy this is still not a thing.
